### PR TITLE
[CSLD-105] Update HD wallets chapter

### PR DESCRIPTION
**Pull request by Martoon-00**
Brought doc up to date with regard to recent features + applied some cosmetic changes

### CODE REVIEW

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 1]
```diff
@@ -9,15 +9,13 @@ visible: true
...
-<!-- @martoon TODO: perhaps this part better fits to protocols section -->
-
 HD wallets is a feature which allows users to derive keys in deterministic way
 from a common seed. Basically, you generate an initial secret key `SK₀` out of a
 random seed. Then you can derive children `SK₀-₀`, `SK₀-₁` out of `SK₀`. From
 these children, you can derive `SK₀-₀-₀`, `SK₀-₀-₁`, `SK₀-₁-₀` and so on
 (derivations for a tree of arbitrary depth).
 
-<!-- Advertisement: for subscripts symbols: https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts -->
+<!-- For subscripts symbols: https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts -->
```
Comment (dniku) [inline _docs/technical/2017-05-14-hd-wallets.md]: If you are on Linux, you can configure XCompose to type in sub- and superscripts directly. Check out this: https://help.ubuntu.com/community/ComposeKey

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 2]
```diff
@@ -32,8 +30,8 @@ available).
 
 Each child is assigned a 4-byte index `i`
 
--   `i > 2³¹ - 1` for **non-hardened** keys
--   `i <= 2³¹ - 1` for **hardened** keys
+-   `i <= 2³¹ - 1` for **non-hardened** keys
```
Comment (dniku) [inline _docs/technical/2017-05-14-hd-wallets.md]: Since we're into Unicode here: `≤`

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 3]
```diff
@@ -183,7 +179,7 @@ with spending password).
 
 ### New storage
 
-Wallet storage is extended to store list of **wallet**. Each wallet corresponds
+Wallet storage is extended to store list of **wallets**. Each wallet corresponds
```
Comment (dniku) [inline _docs/technical/2017-05-14-hd-wallets.md]: _a_ list of wallets

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 4]
```diff
@@ -183,7 +179,7 @@ with spending password).
...
 to single root secret key (backed up by mnemonics and encrypted with spending
```
Comment (dniku) [inline _docs/technical/2017-05-14-hd-wallets.md]: _a_ single key

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 5]
```diff
@@ -215,11 +211,9 @@ User is able to:
 
 -   assign name to **wallets** and **accounts**
 
--   generate arbitrary amount of addresses
+-   generate arbitrary amount of **addresses**
```
Comment (dniku) [inline _docs/technical/2017-05-14-hd-wallets.md]: _an_ arbitrary _number_ of addresses

**Review by dniku — CHANGES_REQUESTED** [review]
Please fix these grammatical issues.

File: _docs/technical/2017-05-14-hd-wallets.md (hunk 1 above)
Comment (Martoon-00) [reply _docs/technical/2017-05-14-hd-wallets.md]: Sugoi!

File: _docs/technical/2017-05-14-hd-wallets.md [hunk 6]
```diff
@@ -54,13 +52,13 @@ For **Hardened** keys:
     A(K) -x pub(K)
     A(K) -x A(child(K, i))
     pub(K) -x pub(child(K, i))
-    priv(K) -> utxo -> tree(K)
+    (priv(K), utxo) -> tree(K)
     priv(K) -> priv(child(K, i))
 
 For **Non-hardened** keys
```
Comment (denisshevchenko) [inline _docs/technical/2017-05-14-hd-wallets.md]: Why "Non-hardened" with capital letter? :-)

**Review by denisshevchenko — CHANGES_REQUESTED** [review]

**Comment by denisshevchenko** [comment]
And please add dot at the end of all complete sentences. :-)

**Comment by Martoon-00** [comment]
Which ones, for instance?

**Review by denisshevchenko — APPROVED** [review]

**Review by staenrey — APPROVED** [review]
grammar checked

**Pull request merged**